.github/workflows/base.yaml

```diff
@@ -0,0 +1,144 @@
+name: Build environment
+
+# Build only on changes to the build environment itself
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - stable
+
+jobs:
+  gcc-x86_64:
+    runs-on: ubuntu-latest
+    env:
+      GCC_VERSION: 12.2.0
+      GCC_ARCH: x86_64
+      GCC_SUFFIX:
+
```

Comment:
Perhaps we can use a matrix, e.g.:
```yaml
jobs:
  archs:
    strategy:
      matrix:
        include:
          - GCC_VERSION: 12.2.0
            GCC_ARCH: x86_64
          - GCC_VERSION: 12.2.0
            GCC_ARCH: arm
          - GCC_VERSION: 12.2.0
            GCC_ARCH: arm64
```

Reply:
yes, we actually should, but right now all other components are disabled.

With only two elements or less I don't see a reason to use a matrix yet

Once we'll support all 3 I think we should add it

Reply:
Can you set it up to use the matrix and then delete the commented-out `gcc-arm`/`gcc-arm64` and just comment out the matrix entries?

Reply:
pray it works :wink: